pkg/handler/v2/rulegroup/request.go

```diff
@@ -193,7 +192,7 @@ func DecodeRuleGroupID(r *http.Request) (string, error) {
 
 func GetRuleGroupNameInData(data []byte) (string, error) {
 	bodyMap := map[string]interface{}{}
-	if err := yaml.UnmarshalStrict(data, &bodyMap); err != nil {
+	if err := yaml.Unmarshal(data, &bodyMap); err != nil {
```

Comment:
`v3.Unmarshal` is not equivalent to `v2.UnmarshalStrict`. Is this intentional? 

https://play.golang.org/p/tb4QNt5jAAk